CO2_Gadget_Menu.h

```diff
@@ -680,7 +680,12 @@ TOGGLE(displayShowPM25, activeDisplayShowPM25, "PM2.5: ", doNothing, noEvent, wr
   ,VALUE("Show", true, doDisplayReverse, enterEvent));
 
 MENU(displayConfigMenu, "Display Config", doNothing, noEvent, wrapStyle
+#ifdef ARDUINO_LILYGO_T_DISPLAY_S3
+  ,FIELD(DisplayBrightness, "Brightness:", "", 0, 15, 1, 1, doSetDisplayBrightness, anyEvent, wrapStyle)
```

Comment:
Shouldn't the minimum be 1 to prevent the user from being stuck with the screen turned off by mistake?

Reply:
Range 0-15 is a logical value to store brigthness level, but 0 is not display off, it's only the lowest level, and 15 the highest.

In menu is possible to define range 1-16 if you prefer.

S3 backlight brightness can't be set to a specific value by default. 
When you "click" the logical TFT_BACKLIGHT button (IO38) the brightness level down from highest intensity (default) by one level, when down to the lowest limit, one "click" more go from lowest to highest level again.

So when you want to decrease 1/16 brightness only one TFT_BACKLIGHT button click is needed, but if you want to increase 1/16 brightness you need to click 15 times.

Reply:
Thank you for the detailed explanation.

Then I think it is better to have a range of 1-16 on the menu. Zero will look like "turned off" to the user.